the-internet/pom.xml

```diff
@@ -53,5 +53,31 @@
 			<version>2.8.6</version>
 		</dependency>
```

Comment:
Input class.
You're clicking the 'up' spinner 100 times but this is actually a test scenario. The Test should determine 'what' is being done, or how many times. The inputNumber method determines how to do the what :)